variables.tf

```diff
@@ -19,4 +19,5 @@ variable "api_cache_default_ttl" {
 }
 variable "mdw_cache_default_ttl" {
   default = 0
-}
\ No newline at end of file
+}
+variable "ch_fqdn" {}
```

Comment:
shouldn't this be something like `domain_name` it seems it's only used in the origin setup